src/Cns.ts

```diff
@@ -202,18 +136,28 @@ export default class Cns extends EthereumNamingService {
     const keyTopics = logs.map(event => event.topics[2]);
     // If there are no NewKey events we want to check the standardRecords
     if (keyTopics.length === 0) {
-      return await this.getStandardRecords(resolverContract, tokenId);
+      return await this.getStandardRecords(tokenId);
     }
-    const keys = await this.callMethod(resolverContract, 'getManyByHash', [
-      keyTopics,
-      tokenId,
-    ]);
-    const keyValues = await this.callMethod(resolverContract, 'getMany', [
+    const [keys, values] = await this.getManyByHash(tokenId, keyTopics);
+    return this.constructRecords(keys, values);
```

Comment:
I would put `constructRecords` to `get` method and return `records: Record<string, string>` from there instead of values. That would remove a little more of code duplication.